__init__.py

```diff
@@ -278,6 +303,15 @@ def start_recording(self, context) -> bool:
                 f'-i ":{addon_prefs.audio_device_darwin}" '
                 f"-framerate {framerate} {self.filepath}"
             )
+        elif os_platform == 'Windows':
+            filepath = chr(34)+(self.filepath)+chr(34)#.replace(chr(92), chr(92)+chr(92))+chr(34)
+            ffmpeg_command = (
+                f"ffmpeg -f dshow "
+                #f'-i ":{addon_prefs.audio_device_windows}" '         
+                f'-i audio="Microphone (Realtek(R) Audio)" '
+                f"-framerate {framerate} {filepath}"
+            )
+            print(ffmpeg_command)
```

Comment:
Please do not use print statements in the code. Consider using log.debug instead.